base/Terminals.jl

```diff
@@ -156,7 +156,11 @@ else
     function hascolor(t::TTYTerminal)
         startswith(t.term_type, "xterm") && return true
         try
-            return success(`tput setaf 0`)
+            @static if Sys.KERNEL == :FreeBSD
```

Comment:
If this doesn't work for FreeBSD, I have a feeling it wouldn't work for Dragonfly and others, so this might be better as
```julia
@static if is_bsd() && !is_apple()
```